public/content/documentation/web/page-level/basic-web-page.md

```diff
@@ -1,44 +1,47 @@
 ## General Notes
 
-Validate your code use HTML validation as the foundation for ensuring your page works for everyone.
+How to test a basic web page
 
 ## Gherkin
 
-### a11y - Web Accessibility Acceptance Criteria
+### #a11y - Web Accessibility Acceptance Criteria
 
 How to test a basic web page
 
 GIVEN THAT I am on a page with a basic web page
 
 1. Keyboard for mobile & desktop
 
-   &mdash; EN I use the keyboard to open a new web page I SEE the page has a unique logical title in the browser tab
+   - WHEN I use the keyboard to open a new web page I SEE the page has a unique logical title in the browser tab
 
 2. Desktop screenreader
 
-   &mdash; EN I use a desktop screenreader (NVDA, JAWS, VoiceOver) AND
-   &mdash; use the keyboard to open a new web page
-   &mdash; HEAR The page has a unique logical title in the browser tab
-   &mdash; HEAR Major landmarks are discoverable with screenreader shortcuts: header/banner, navigation, main and footer/content info landmarks
+   - WHEN I use a desktop screenreader (NVDA, JAWS, VoiceOver) AND 
+   - I use the keyboard to open a new web page
+      - I HEAR The page has a unique logical title in the browser tab
+      - I HEAR Major landmarks are discoverable with screenreader shortcuts: header/banner, navigation, main and footer/content info landmarks
 
 3. Mobile screenreader
 
-   &mdash; EN I use a mobile screenreader (Talkback, VoiceOver) AND
-   &mdash; swipe to enter from the web browser tabs
-   &mdash; HEAR The page has a unique logical title in the browser tab
-   &mdash; HEAR Major landmarks are discoverable with screenreader shortcuts: header/banner, navigation, main and footer/content info landmarks
-   &mdash; EN when I change orientations I HEAR content is accessible in landscape or portrait orientation
+   - WHEN I use a mobile screenreader (Talkback, VoiceOver) AND
+   - I swipe to enter from the web browser tabs
+      - I HEAR The page has a unique logical title in the browser tab
+      - I HEAR Major landmarks are discoverable with screenreader shortcuts: header/banner, navigation, main and footer/content info landmarks
+   - THEN when I change orientations I HEAR content is accessible in landscape or portrait orientation
 
 4. Device OS settings
-   &mdash; EN I use zoom/pinch THEN I see text can resize up to 200% without losing information
+   - WHEN I use zoom/pinch THEN I see text can resize up to 200% without losing information
 
 Full information: https://www.magentaa11y.com/#/web-criteria/page-level/basic-web-page
 
+## Validate your code
+Use [HTML validation](https://validator.w3.org/nu/) as the foundation for ensuring your page works for everyone.
+
 ## Code Examples
 
 ### Declare a language
 
-This affects the screenreader pronunciation.
+This aids a screen reader in reading a content in the proper language.
 
 ```html
 <html lang="en"></html>
```

Comment:
I can't add this comment directly to where it should be, but the following edits are needed under "Give your page a unique title":
1. "Each page.." should be a bullet point
2. "This includes.." should be indented on level under the above
3. "if" in the above bullet point should be italicized
4. I believe "URl" in the same bullet point should be URL (with a capital L, which is a miss on our original site, too). 

If these should be a new/separate commit, happy to tackle these updates.

Reply:
for 1. I thought it looked nicer as a sentence so I thought I'd leave it!

Reply:
same idea for 2.

Reply:
got it on 3 and 4!